source/_integrations/verisure.markdown

```diff
@@ -138,3 +138,16 @@ automation:
 | disable_autolock | Disables autolock function for a specific lock. |
 | enable_autolock | Enables autolock function for a specific lock. |
 | smartcam_capture | Capture a new image from a specific smartcam. |
+
+## 2 Factor Authentication Workaround
```

Comment:
This is a prerequisite, let's put it above the configuration section and call it as such.

Reply:
I've moved it now and renamed the title. Do you think the location and new title is ok?